supporting-blog-content/alternative-approach-for-parsing-pdfs-in-rag/README.md

```diff
@@ -0,0 +1,19 @@
+
+# PDF Parsing - Table Extraction
+
+ Python notebook demonstrates an alternative approach to parsing PDFs, particularly focusing on extracting and converting tables into a format suitable for search applications such as Retrieval-Augmented Generation (RAG). The notebook leverages Azure OpenAI to process and convert table data from PDFs into plain text for better searchability and indexing.
+
+## Features
+- **PDF Table Extraction**: The notebook identifies and parses tables from PDFs.
+- **LLM Integration**: Calls Azure OpenAI models to provide a text representation of the extracted tables.
+- **Search Optimization**: The parsed table data is processed into a format that can be more easily indexed and searched in Elasticsearch or other vector-based search systems.
+  
+## Getting Started
+
+### Prerequisites
+- Python 3.x
+
+
+## Example Use Case
```

Comment:
Can you show the sample input (fake OpenAI endpoint, fake OpenAI key, output file directory, pdf name) for the use case in the notebook?

Reply:
Also a link to the PDF that we will be sending to OpenAI can help the reader see the utility of creating text from tables. An screencap of the table would also be sufficient.

Reply:
@justincastilla  the table image is in the article.  I have added the other suggestions.